store/src/java/com/zimbra/cs/datasource/imap/MessageChanges.java

```diff
@@ -90,6 +97,28 @@ private MessageChanges findChanges(int changeId) throws ServiceException {
         return this;
     }
 
+    private MessageChanges findChangesForDraft(int changeId, int draftFolderId) throws ServiceException {
+        // Find modified items since specified change id for draft folder
```

Comment:
use small case for comments to start with here and other places

Reply:
made all single-line comments as small case in all my changes

Reply:
You don't need to change anything, but I usually aim for consistency with the rest of the file.

Reply:
Yes they were capital, changing it back to capital